Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1

```diff
@@ -1,108 +1,168 @@
 #>
 
+[CmdletBinding()]
 param (
-    [Parameter(
-        Mandatory, 
-        Position = 1
-    )]
-    [string]
-    $Commit,
-    [Parameter(
-        Mandatory, 
-        Position = 2
-    )]
-    [string]
-    $Build,
-    [Parameter(
-        Mandatory,
-        Position = 3
-    )]
-    [string]
-    $APIKey,
-    [Parameter(
-        Position = 4
-    )]
+    [Parameter(Mandatory, Position = 1)]
+    [string]$Commit,
+
+    [Parameter(Position = 2)]
+    [string]$Build,
+
+    [Parameter(Mandatory, Position = 3)]
+    [string]$AppAPIKey,
+
+    [Parameter(Position = 4)]
     [System.IO.FileInfo]
     $OutputFile = ".$([System.IO.Path]::DirectorySeparatorChar)supported-indexers.md",
-    [Parameter(
-        Position = 5
-    )]
-    [uri]
-    $BaseURL = "http://localhost:9696"
-)
-
-$url=($BaseURL.ToString().TrimEnd("/"))+'/api/v1/Indexer/schema'
-$headers=@{'X-Api-Key'=$APIKey}
-$R= Invoke-WebRequest -Uri $url -Headers $headers -ContentType 'application/json' -Method Get
-$exp_indexername= { IF ($_.IndexerUrls) {'['+$_.name+']('+($_.IndexerUrls[0])+')'+'{#'+$_.infoLink.Replace('https://wiki.servarr.com/prowlarr/supported-indexers#','')+'}'} Else {$_.name+'{#'+$_.infoLink.Replace('https://wiki.servarr.com/prowlarr/supported-indexers#','')+'}'}}
-$indexerobject= $R.Content | ConvertFrom-Json
-$tblobj= $indexerobject | Sort-Object -Property 'name'
-
-#$tbl_PubUse= $tblobj | Where-Object {($_.privacy -eq "public") -and ($_.protocol -eq "usenet")} | `
-#         Select-Object @{Name = 'Indexer'; Expression =$exp_indexername}, `
-#         @{Name = 'Language'; Expression = {$_.language}}, `
-#         @{Name = 'Description'; Expression = {$_.description}}
-
-$tbl_PrvUse= $tblobj | Where-Object {($_.privacy -CIn "private" -and $_.protocol -eq "usenet")} | `
-         Select-Object @{Name = 'Indexer'; Expression =$exp_indexername}, `
-         @{Name = 'Language'; Expression = {$_.language}}, `
-         @{Name = 'Description'; Expression = {$_.description}}
-
-$tbl_PubTor= $tblobj | Where-Object {($_.privacy -eq "public") -and ($_.protocol -eq "torrent")} | `
-         Select-Object @{Name = 'Indexer'; Expression =$exp_indexername}, `
-         @{Name = 'Language'; Expression = {$_.language}}, `
-         @{Name = 'Description'; Expression = {$_.description}}
-        
+
+    [Parameter(Position = 5)]
+    [uri]$AppBaseURL = 'http://localhost:9696'
+),
+
+## Gather Inputs & Variables
+### User Inputs
+#### Convert Params to match vars
+$app_baseUrl = $AppBaseURL
+$app_apikey = $AppAPIKey
+####
+
+## Start Variables
+### Application Details
+$app_api_version = 'v1'
+$app_api_path = '/api/'
+$app_api_endpoint_version = '/system/status'
+$app_api_endpoint_indexer = '/indexer/schema'
+$headers = @{'X-Api-Key' = $app_apikey }
+#### Github App Info
+$gh_app_org = 'Prowlarr'
+$gh_app_repo = 'Prowlarr'
+### Wiki Details
+$wiki_link = 'https://wiki.servarr.com'
+$wiki_app_path = '/prowlarr'
```

Comment:
These are only used in one place - why use variables?

Reply:
flexibility/futureproof and this way all the specifics are needed in one place rather than buried down below